.github/workflows/deploy.yml

```diff
@@ -0,0 +1,125 @@
+# THIS FILE IS PART OF THE CYLC SUITE ENGINE.
+# Copyright (C) 2008-2019 NIWA & British Crown (Met Office) & Contributors.
```

Comment:
Minor niggle - copyright year out of date

Reply:
Well spotted.